### PR TITLE
Switching holoviews to noarch

### PR DESCRIPTION
**Pull request by jlstevens**
Now that development builds are configured, I would like to use them to see whether we can switch to noarch, addressing issue #16.

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

Note: Needed after changing to `noarch`. 😉

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ source:
   git_depth: 60
 
 build:
+  noarch: python
   number: 0
```
Comment (jakirkham) [inline recipe/meta.yaml]: Also would recommend bumping the build number.

Comment (jlstevens) [reply recipe/meta.yaml]: Sure!

**Comment by jlstevens** [comment]
Seeing +3 lines and −191 lines after rerendering just shows why noarch is a good idea for everyone (where applicable)!

**Comment by jlstevens** [comment]
Tests are green. I hope to see this merged soon so I can go ahead and test it myself...

**Comment by philippjfr** [comment]
Looks good to me. I'll leave merging to you @jakirkham, since you'll know better if everything looks good.

**Comment by jakirkham** [comment]
LGTM. Feel free to merge.

At some point, would suggest [reading this short section in the docs]( https://conda-forge.org/docs/meta.html#building-noarch-packages ) just for familiarity.

**Comment by philippjfr** [comment]
Okay I'll merge. Reading the docs is a good suggestion in general, so I'll just read the whole thing.

**Pull request merged**